app/src/main/java/com/greenfox/opal/gitinder/service/GithubApiService.java

```diff
@@ -0,0 +1,17 @@
+package com.greenfox.opal.gitinder.service;
+
+import com.greenfox.opal.gitinder.model.response.GithubUser;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Header;
+
+/**
+ * Created by Nagy Dóra on 2017.06.26..
+ */
+
+public interface GithubApiService {
+
+  @GET("/user")
+  Call<GithubUser> getUser(@Header("X-GiTinder-token") String token);
```

Comment:
@balintvecsey `X-GiTinder-token` possibly is not needed when calling the Github api, they need their `access_token` though in some form, check the github api documentation for that (or retrofit example, because thats basically it)